group.yml

```diff
@@ -13,6 +13,9 @@ vars:
   CVES: None
   RHCOS_EL_MAJOR: 9
   RHCOS_EL_MINOR: 4
+  # It is not decided yet from which OCP version on a new release version will have the format i.e. 4.18.0-0
+  # Until then the NEW_RELEASE_VERSION will have the value False
+  NEW_RELEASE_VERSION: false
```

Comment:
do not think this variable exists for string replacement

Reply:
resoved with the latest update